rewrite-gradle/src/main/java/org/openrewrite/gradle/AddDelegatesToGradleApi.java

```diff
@@ -79,7 +79,7 @@ public J.MethodDeclaration visitMethodDeclaration(J.MethodDeclaration method, Ex
                         return param;
                     }
                     // Construct a matcher that will identify methods identical to this one except they accept Action instead of Closure
-                    MethodMatcher matcher = new MethodMatcher(method.getMethodType().withParameterTypes(
+                    MethodMatcher matcher = MethodMatcher.create(method.getMethodType().withParameterTypes(
```

Comment:
A minor suggestion here, I'd rename `create` to `build` to reflect our other APIs.

Reply:
I'm easy, happy to make that change. In my experience `build` implies a 'completion' or 'finalization' of some kind of builder pattern style API. This isn't really the builder pattern here, so i don't know if I'd use the `build` method name.

I'm happy to make the change if you still think that `build` is better. But it's not the name I'd expect as an API user.